packages/data-explorer-ui/src/components/Export/components/DownloadCurlCommand/components/DownloadCurlCommandNotStarted/downloadCurlCommandNotStarted.tsx

```diff
@@ -24,6 +25,7 @@ export interface DownloadCurlCommandNotStartedProps {
 }
 
 export const DownloadCurlCommandNotStarted = ({
+  disabled,
```

Comment:
Added `disabled` prop here and in other export methods to disable the export e.g. when Azul is indexing.